projecttacoma/measure-repository
# Updated loadBundles script for connectathon and add QICore ModelInfo

# Summary

Latest connectathon ecqm-content-qicore-2024 bundles still have some issues with references. This adds the loadBundles script from the previous connectathon and make it fix the current issues with the bundles references, specifically with the library relatedArtifact references.

Also adds QICore ModelInfo library that is now referenced by measures.

## New behavior

`npm run loadBundles` in service project loads bundles in the `ecqm-content-qicore-2024` repo into a running measure repository.

`npm run db:setup` now inserts the QICore ModelInfo library when setting up the database.

## Code changes

New script `script/loadBundles`.

# Testing guidance

- `npm run db:reset`
- Start up MRS and front end. Ensure FHIR ModelInfo and QICore ModelInfo exist in the library list
- Have the latest `ecqm-content-qicore-2024` repo checked out adjacent to this repository.
- `npm run loadBundles`
- This script will output a lot of info about what it does. Duplicate shared libraries will report uploading with 400 errors.
- Ensure that measures appear in repository and can have $data-requirements executed on them in the UI and are able to show dependencies.
